lib/ansible/modules/storage/zfs/zpool_facts.py

```diff
@@ -157,11 +160,15 @@ def get_facts(self):
         cmd = [self.module.get_bin_path('zpool')]
 
         cmd.append('get')
-        cmd.append('-H')
+        if not self.is_sunos:
+            # solaris 11 does not support -H
```

Comment:
```suggestion
            # solaris 11 does not support -H
```

i don't get it how solaris 11 does not support -H , being on my solaris 11.4 box:

```
root@deimos:sysadmin# uname -v
11.4.1.4.0
root@deimos:sysadmin# zpool get -H health amstank
amstank health  ONLINE  -
```